blockmanager.go

```diff
@@ -1358,16 +1358,21 @@ func (b *blockManager) detectBadPeers(headers map[string]*wire.MsgCFHeaders,
 	)
 	return resolveFilterMismatchFromBlock(
 		block.MsgBlock(), fType, filtersFromPeers,
+
+		// We'll need half of our peers to agree on filters.
+		(len(filtersFromPeers)+1)/2,
```

Comment:
We'd want at least a minimum of 3 peers in order to detect a majority, so I think the minimum threshold should be 2.

Reply:
Done